pkg/operation/common/types.go

```diff
@@ -75,6 +75,24 @@ const (
 	// AlertManagerPrefix is a constant for a prefix used for the AlertManager instance.
 	AlertManagerPrefix = "au"
 
+	// LokiPrefix is a constant for a prefix used for the Loki instance.
+	LokiPrefix = "l"
```

Comment:
Why the prefix is just one letter?

Reply:
I think this was done to prevent running into length restrictions (see others above (prometheus, alertmanager, ..))